src/Onboarding/EulaModal.tsx

```diff
@@ -128,6 +130,16 @@ const EulaModal: FunctionComponent<EulaModalProps> = ({
 
   const canContinue = boxChecked
 
+  const handleOnPressContinue = () => {
+    setModalVisibility(false)
+    continueFunction()
```

Comment:
I know this wanst introduced by you here. but `continueFunction` is a pretty unhelpful name. what do you think about renaming it here to something like: `onPressModalContinue` or similiar?